### ISSUE TITLE
Add tooltip to system tray

### ISSUE THREAD

**Issue by robinpersson** [fyne-io/fyne#5081]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

Yes, when hover the system tray icon in windows the tooltip is empty. There is no way to set the tooltip text on the system tray

### Is it possible to construct a solution with the existing API?

No not that I know

### Describe the solution you'd like to see.

Add support for adding a tooltip text to the system tray

**Comment by andydotxyz** [comment]
I would recommend trying `systray.SetTitle`

**Comment by andydotxyz** [comment]
System tray setup is time critical in some ways that you should not have to worry about.
Once tooltips are in Fyne this will be handled for you - but for now don't use the systray APIs until the app is launched.

The following will work:

```go
a.Lifecycle().SetOnStarted(func() {
	systray.SetTooltip("test tooltip")
})
```

**Comment by udelledo** [comment]
Does this mean that I won't be able to dynamically update the tooltip while
the application is running?

Il giorno ven 15 nov 2024 alle ore 22:08 Andy Williams <
***@***.***> ha scritto:

**Comment by andydotxyz** [comment]
No, you can set it any time /after/ the app has finished launching. Simulating later is not a problem.
